### ISSUE TITLE
doesn't detect nearby devices

### ISSUE THREAD

**Issue by fahimscirex** [Martichou/rquickshare#84]
rquickshare works perfectly from Phone-to-PC, but it never detects nearby available devices thus can't send any files.

Network Chip: RTL8821CE
OS: Archlinux with Hyprland

**Comment by Martichou** [comment]
Android doesn't advertise it's MDNS service all the times. 
So you need to practively "trigger" that using a method as explained in the README: https://github.com/Martichou/rquickshare?tab=readme-ov-file#my-laptop-doesnt-see-my-android-device. 

This is until I fix the Bluetooth advertisment (but it can take some time as I don't have a Bluetooth chip recent enough for that).

**Comment by fahimscirex** [comment]
couldn't find those two intents on my Samsung device, can you help me with that?

**Comment by Martichou** [comment]
If those two are not present then I think there's no way.
I saw that Google is thinking of adding a QR code to the QuickShare protocol to discover devices faster, etc. That would be a nice workaround but we'll have to wait for google to push it and I'll have to see what we can do with those QR.

Tldr: I'm sorry for you but it seems Samsung removed the only two workaround intents I know.

**Comment by berkes** [comment]
Google now has a "Use QR code" feature. This works as follows: 

We can see that:

* The accepting side (i.e. rquickshare) has to scan the QR code.
* The QR code leads to a web URL https://quickshare.google/qrcode#key=[redacted]

Should this be made a separate feature request?

![Image](https://github.com/user-attachments/assets/e07d7010-7eee-44e9-86f7-ef13a456ea08)

**Comment by fahimscirex** [comment]
I'm assuming this if you connect using QR the file will be transferred over the internet not through you local network, so it doesn't really help in that case.

**Comment by Martichou** [comment]
That would be really cool, but afaik very difficult as we have to rely on Google servers (and RQuickShare being not official, have no access to official servers). 

So cool that Google added this, but not usable for us.
